packages/abi-gen/package.json

```diff
@@ -29,12 +31,19 @@
     "dependencies": {
         "@0xproject/typescript-typings": "^0.4.1",
         "@0xproject/utils": "^0.7.1",
-        "ethereum-types": "^0.0.1",
+        "@types/sleep": "^0.0.7",
+        "@types/tmp": "^0.0.33",
```

Comment:
Types that aren't part of the exported, public interface, they should be `devDependencies`. Please move them there.